navigator/backend/src/test/scala/com/digitalasset/navigator/backend/IntegrationTest.scala

```diff
@@ -48,53 +75,146 @@ class IntegrationTest
         commandClient = CommandClientConfiguration.default,
       ),
     )
-    val fa = for {
+
+    // TODO https://github.com/digital-asset/daml/issues/12663 participant user management: Emulating no-pagination
+    def listAllUsers(client: LedgerClient) =
+      client.userManagementClient.listUsers(pageToken = "", pageSize = 10000).map {
+        case (users, _) => users.toList
+      }
```

Comment:
Eliminate your `TODO`s with this simple trick that people in the Ivory Tower don't want you to know.

https://github.com/digital-asset/daml/blob/9c35fa286e04a42dedac073202d58dd1219fca55/ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala#L495-L503